CMSgov/onemac
# adding info text to file download lists

Story: https://qmacbis.atlassian.net/browse/OY2-13923
Endpoint: https://d1nn8bs7pz4bgh.cloudfront.net/

### Details

As a OneMAC User, I need a warning message on the package details page that the attachments displayed may not be final approved documents so that I do not assume that the OneMAC documents are always approved.

### Changes

- Added new input property to FileList component for informational text
- reorganized FileList component to move the download all button below the information text per HCD

### Test Plan

1. Login as any user with view privileges 
2. Open a SPA/Waiver
3. Verify the informational text exists between the header "Base Supporting Documentation" and the Download All button
